.well-known/funding-manifest-urls

```diff
@@ -0,0 +1 @@
+https://opencrvs.org/funding.json
```

Comment:
Should this file be in files gateway / client etc. web server exposes? .well-known directory to me implies a HTTP context